Sources/MapFilter/MapFilterViewController.swift

```diff
@@ -41,25 +41,7 @@ public class MapFilterViewController: FilterViewController {
     public override func viewDidLoad() {
         super.viewDidLoad()
         setup()
-        showApplyButton(true, animated: false)
-    }
-
-    public override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        guard let currentSelection = selectionDataSource.geoValue(for: filterInfo) else {
-            return
-        }
-        mapFilterView?.setInitialSelection(latitude: currentSelection.latitude, longitude: currentSelection.longitude, radius: currentSelection.radius, locationName: currentSelection.locationName)
-    }
-
-    public override func viewWillDisappear(_ animated: Bool) {
-        super.viewWillDisappear(animated)
-
-        // TODO: This should be done when apply or back is pressed
-        guard let mapFilterView = mapFilterView, let coordinate = mapFilterViewManager?.centerCoordinate else {
-            return
-        }
-        selectionDataSource.setValue(latitude: coordinate.latitude, longitude: coordinate.longitude, radius: mapFilterView.currentRadius, locationName: nil, for: filterInfo)
+        showApplyButton(true)
```

Comment:
You might want to use `showApplyButton(true, animated: false)` as before.

Reply:
Ah, I see now that it defaults to `animated = true`